.github/CODEOWNERS

```diff
@@ -43,9 +43,11 @@
 /model/user-agent/                @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
 
 # System semantic conventions
+/docs/process/                    @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers @open-telemetry/semconv-security-approvers
```

Comment:
@open-telemetry/semconv-system-approvers I hope you don't mind beling listed as codeowners for `process`. I'm surprised you folks aren't.